features/providers/complete_and_check_means_answers.feature

```diff
@@ -5,54 +5,100 @@ Feature: Completing and checking means answers backwards and forwards
+    When I select "None of the above"
+    And I click 'Save and continue'
+
+    Then I should be on a page showing "Does your client receive student finance?"
+
+    When I choose "No"
+    And I click 'Save and continue'
+    Then I should be on the 'identify_types_of_outgoing' page showing "Which payments does your client make?"
+    Then I select "Childcare"
+    And I click 'Save and continue'
+    Then I should be on the 'cash_outgoing' page showing "Select payments your client makes in cash"
+
+    When I select "None of the above"
+    And I click 'Save and continue'
+    Then I should be on the 'income_summary' page showing "Sort your client's income into categories"
+
+    When I click the first link 'View statements and add transactions'
+    Then I should be on a page showing 'Select benefits payments'
+    When I select the first checkbox
+    And I click 'Save and continue'
+    And I click 'Save and continue'
+    And I click 'Save and continue'
+    Then I should be on the 'outgoings_summary' page showing "Sort your client's regular payments into categories"
+
+    When I click the first link 'View statements and add transactions'
+    Then I should be on a page showing 'Select childcare payments'
+    When I select the first checkbox
+    And I click 'Save and continue'
     Then I click 'Save and continue'
+
+    When I click 'Save and continue'
     Then I should be on the 'has_dependants' page showing "Does your client have any dependants?"
-    Then I choose "No"
-    Then I click 'Save and continue'
-    Then I should be on a page showing "Your client's outgoings"
-    Then I choose "No"
-    Then I click 'Save and continue'
-    Then I should be on a page showing "Which payments does your client make?"
-    Then I select 'Housing payments'
-    Then I select 'Payments towards legal aid in a criminal case'
-    Then I click 'Save and continue'
-    Then I should be on a page showing "Sort your client's regular payments into categories"
-    Then I should be on a page showing "Housing payments"
-    Then I should be on a page showing "Payments towards legal aid in a criminal case"
-    Then I click link 'Add another type of regular payment'
-    Then I select 'My client makes none of these payments'
-    Then I click 'Save and continue'
-    Then I should be on a page showing "Sort your client's regular payments into categories"
-    Then I should be on a page not showing "Housing payments"
-    Then I should be on a page not showing "Payments towards legal aid in a criminal case"
-    Then I click 'Save and continue'
+    When I choose "No"
+    And I click 'Save and continue'
     Then I should be on a page showing "Does your client own the home that they live in?"
 
   @javascript @vcr
   Scenario: I navigate to the Check your answers page and then add some outgoing transaction types
     Given The means questions have been answered by the applicant
```

Comment:
means question are no longer answered byt eh applicant, right? in which case this step needs to be replaced I am guessing?!